### PR TITLE
feat: [IOBP-170] Add multiple pagoPA barcode choice screen

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the screen which allow to choose from multiple decoded pagoPA barcodes

https://github.com/pagopa/io-app/assets/6160324/889e00d8-0eda-482d-b8d1-74741cc724eb

## List of changes proposed in this pull request
- Added `WalletPaymentBarcodeChoiceScreen.tsx`
- Added logic to handle multiple barcodes on `WalletPaymentBarcodeScanScreen.tsx` and `BarcodeScanScreen.tsx`

## How to test
With the `io-dev-server`, try to scan a document with multiple pagoPA barcodes.

### CODE REVIEW

**Review by Hantex9 — CHANGES_REQUESTED** [review]
There is a strange behavior when I upload a file with more than one valid pagoPA qr code.

I uploaded a file with 2 valid pagoPA codes and it navigates correctly to the choosing screen but inside the list is rendered the same amount and code of the last qrcode repeated twice.

I try to attach a screenshot, the code and details are omitted but I can assure you that they are the same (look at the amount) even if a notice is of 141,87€ and the other 102,43€

File: ts/features/walletV3/payment/components/PaymentNoticeListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  return (
+    <PressableListItemBase onPress={onPress}>
+      <View style={{ flexGrow: 1 }}>
+        <NewLabelSmall>{paymentNoticeNumber}</NewLabelSmall>
+        <LabelSmall weight="Regular" color="grey-700">
+          {organizationFiscalCode}
+        </LabelSmall>
+      </View>
+      <H6 color="blueIO-450">{amountString}</H6>
```
[the remark this answers is not in the document]
Comment (mastro993) [reply ts/features/walletV3/payment/components/PaymentNoticeListItem.tsx]: Fixed in https://github.com/pagopa/io-app/pull/4915/commits/51edd07e7886e32089601e619258b34dbed5c0f0

**Comment by mastro993** [comment]
> a36bc67

Fixed in https://github.com/pagopa/io-app/pull/4915/commits/a36bc677ce40fb0f74e3668b36e6d55fecb58712

**Comment by mastro993** [comment]
> There is a strange behavior when I upload a file with more than one valid pagoPA qr code.
> 
> I uploaded a file with 2 valid pagoPA codes and it navigates correctly to the choosing screen but inside the list is rendered the same amount and code of the last qrcode repeated twice.
> 
> I try to attach a screenshot, the code and details are omitted but I can assure you that they are the same (look at the amount) even if a notice is of 141,87€ and the other 102,43€
> 
> 

Fixed in https://github.com/pagopa/io-app/pull/4915/commits/a36bc677ce40fb0f74e3668b36e6d55fecb58712

**Review by Hantex9 — APPROVED** [review]
LGTM! 👏

**Pull request merged**